hummingbot/market/ddex/ddex_api_order_book_data_source.py

```diff
@@ -89,11 +89,11 @@ def __init__(self, trading_pairs: Optional[List[str]] = None):
             all_markets: pd.DataFrame = pd.DataFrame.from_records(data=ticker_data,
                                                                   index="marketId")
 
-            dai_to_eth_price: float = float(all_markets.loc["SAI-WETH"].price)
+            sai_to_weth_price: float = float(all_markets.loc["WETH-SAI"].price)
             weth_to_usd_price: float = float(all_markets.loc["WETH-TUSD"].price)
             usd_volume: float = [
                 (
-                    quoteVolume * dai_to_eth_price * weth_to_usd_price if trading_pair.endswith("SAI") else
+                    quoteVolume * sai_to_weth_price * weth_to_usd_price if trading_pair.endswith("SAI") else
```

Comment:
do we still need to multiply `weth_to_usd_price`?

Reply:
Yea notice that. Push a new commit to resolve this. 
Also I chose to use the `SAI-NUSD` trading pair